devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -41,6 +41,34 @@ public abstract class Ssd13xx : IDisposable
             0xaf        // turn display on 0xaf
         };
 
+        /// <summary>
+        /// Sequence of bytes that should be sent to a 128x64 OLED display to setup the device.
+        /// First byte is the command byte 0x00.
+        /// </summary>
+        private readonly byte[] _oled64x128Init =
+        {
+            0x00,       // is command
+            0xd5,       // 1 clk div
+            0xae,       // turn display off
+            0xd5, 0x80, // set display clock divide ratio/oscillator,  set ratio = 0x80
+            0xa8, 0x3f, // set multiplex ratio 0x00-0x3f        
+            0xd3, 0x1f, // set display offset 0x00-0x3f, no offset = 0x00
+            0x40 | 0x0, // set display start line 0x40-0x7F
+            0x8d, 0x14, // set charge pump,  enable  = 0x14  disable = 0x10
+            0x20, 0x00, // 0x20 set memory address mode,  0x0 = horizontal addressing mode
+            0xa0 | 0x01, // set segment re-map
+            // 0xa1 | 0x01, // set segment re-map
```

Comment:
if this line is not needed then remove it otherwise please add a little comment on why it should be kept.

Reply:
Yeah.  Left there while I was testing.  It's now removed.